packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,27 @@
+# Best Practices
+The Flat select option should be used for single choice selection.
+
+The width of each of the options should be the same, that way we keep everything equal and balanced, we don't want any of the options feel more important than the other.
+
+Limit the number of options, a good rule of thumb is to have no more than five.
+
+Don't use Flat select when the effect is not immediate, consider using dropdown menu, radio buttons or checkboxes instead. Users are expect to have immediate feedback on their actions when using Flat select.
+
+If used as view switcher (tabs), make sure it makes sense hierarchically and that tabs cannot be used.
+
+## Labeling
+- Keep titles short, preferably **under three words**.
+- Write for scannability
```

Comment:
I saw that in your other pr and i'm not sure what it means? :eyes:

Reply:
@aiheon Its about the labels in the button, or am I missing the question? 😅